### PR TITLE
DFK node build from source

### PR DESCRIPTION
**Pull request by aaronbuchwald**
This PR updates the DFK Node tutorial to build AvalancheGo directly from source to make it simpler to manage for this tutorial.

### CODE REVIEW

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md [hunk 1]
```diff
@@ -6,46 +6,56 @@ We recently deployed the DFK subnet on Avalanche mainnet and successfully launch
...
 
 ### Updating Config File
 
 You can skip this section if you want to pass whitelisted subnets through command-line flags.
 
-You need to edit your existing config file for your node. If you are running a node using [installer script](../../nodes-and-staking/set-up-node-with-installer), it can be found in `~/.avalanchego/configs/node.json`. If you are not using any config file, then you can make it anywhere on your system. Update this JSON file with the whitelisted subnet. 
+You need to edit your existing config file for your node. In this tutorial we will create a config file at: `~/.avalanchego/config.json`. If you are not using any config file, then you can create a JSON file anywhere on your file system and specify the config file when running your AvalancheGo node at the command line.
```
Comment (aaronbuchwald) [inline docs/build/tutorials/platform/subnets/setup-dfk-node.md:56]: clean up

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md [hunk 2]
```diff
@@ -54,18 +64,20 @@ You need to edit your existing config file for your node. If you are running a n
...
-If you want to pass the whitelisted subnets through the command line flag. You can append the other flags or even the `--config-file` flag as well, according to your need.
+### Running the Node
+
+First, make sure to shut down your node in case it is still running. Then, we will navigate back into the AvalancheGo directory and launch the node.
+
+If you went through the steps to set up a config file, then you can launch your node by specifying the config file on the command line:
 
 ```bash
-./avalanchego/build/avalanchego --whitelisted-subnets mDV3QWRXfwgKUWb9sggkv4vQxAQR4y2CyKrt5pLZ5SzQ7EHBv
+./avalanchego/build/avalanchego --config-file ~/.avalanchego/config.json
```
Comment (aaronbuchwald) [inline docs/build/tutorials/platform/subnets/setup-dfk-node.md:76]: fix: cd into avalanchego directory and run binary.

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md [hunk 3]
```diff
@@ -54,18 +64,20 @@ You need to edit your existing config file for your node. If you are running a n
...
 ```
 
-If you want to pass the whitelisted subnets through the config file.
+If you want to pass the whitelisted subnets through the command line flag. You can append the other flags or even the `--config-file` flag as well, according to your need.
 
 ```bash
-./avalanchego/build/avalanchego --config-file ./config.json
+./avalanchego/build/avalanchego --whitelisted-subnets Vn3aX6hNRstj5VHHm63TCgPNaeGnRSqCYXQqemSqDd2TQH4qJ
```
Comment (aaronbuchwald) [inline docs/build/tutorials/platform/subnets/setup-dfk-node.md:82]: same comment

**Review by yulin-dong — APPROVED** [review]

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md [hunk 4]
```diff
@@ -6,46 +6,56 @@ We recently deployed the DFK subnet on Avalanche mainnet and successfully launch
...
+./scripts/build.sh $GOPATH/src/github.com/ava-labs/avalanchego/build/plugins/mDV3QWRXfwgKUWb9sggkv4vQxAQR4y2CyKrt5pLZ5SzQ7EHBv
 ```
 
+The long string `mDV3QWRXfwgKUWb9sggkv4vQxAQR4y2CyKrt5pLZ5SzQ7EHBv` is the CB58 encoded VMID of the DFK subnet-evm. AvalancheGo will use the name of this file to determine what VMs are available to run from the `plugins` directory.
+
 ## Whitelisting DFK Subnet and Restarting the Node
 
-To start syncing the state with the chains on a subnet, we need to whitelist the subnet and restart the node. You can either add the whitelisted subnets in a config file or pass them through command-line options.
+AvalancheGo will only validate the primary network by default. In order to add the DFK Subnet, we will need to add the DFK Subnet ID to the set of whitelisted subnets in the node's config file or pass it through the command line options of the node. Once the node's config file has been updated, you will need to restart the Avalanche node (if running), or start the node to begin syncing both the Primary Network and the DFK Subnet with the new parameters as well as with the plugin binary in the correct place.
```
Comment (aaronbuchwald) [inline docs/build/tutorials/platform/subnets/setup-dfk-node.md:50]: switch from we to you throughout

Comment (aaronbuchwald) [reply docs/build/tutorials/platform/subnets/setup-dfk-node.md:50]: also clean up end of this sentence it's a run on

File: docs/build/tutorials/platform/subnets/setup-dfk-node.md [hunk 5]
```diff
@@ -6,46 +6,56 @@ We recently deployed the DFK subnet on Avalanche mainnet and successfully launch
...
+git clone https://github.com/DefiKingdoms/subnet-evm
+cd subnet-evm
 ```
 
-The above command will clone the `subnet-evm` repository and build its binary inside the `subnet-evm/build` directory. The name of the binary will be the same as its vmID.
-
-## Move Binary to Plugins Directory
-
-Now we need to move the built binary to the `avalanchego/build/plugins` directory. Run the following command to move the binary to the `plugins` folder. Make sure to use the actual location of your `avalanchego` source and other relative paths. This command assumes we are in the `dfk-subnet` folder and the `avalanchego` source is in its sibling (.\./) folder.
+Now that we are in the DeFiKingdoms/subnet-evm repository, we will build the binary and place it directly into the AvalancheGo `build/plugins` directory by passing in the location on the path we want to place the plugin binary (which was created when building AvalancheGo).
```
Comment (aaronbuchwald) [inline docs/build/tutorials/platform/subnets/setup-dfk-node.md:40]: simplify end of sentence

File: docs/build/tutorials/nodes-and-staking/run-avalanche-node.md [hunk 6]
```diff
@@ -51,28 +51,31 @@ If you want to build the node from source, you're first going to need to install
 
 Run `go version`. **It should be 1.17.9 or above.** Run `echo $GOPATH`. **It should not be empty.**
 
-Download the AvalancheGo repository:
+Download the AvalancheGo repository into your `$GOPATH`:
```
Comment (patrick-ogrady) [inline docs/build/tutorials/nodes-and-staking/run-avalanche-node.md:54]: I think including `$GOPATH` here will confuse people. I'd just keep it as clone.

**Pull request merged**